### ISSUE TITLE
Red and pink colors in dark mode diffs have low contrast

### ISSUE THREAD

**Issue by outofambit** [desktop/desktop#6575]
Red and pink colors have a contrast of 2.6:1. Does not meet the contrast minimum of 4.5:1 for text.

<img width="789" alt="tdna_gh_desk_mac_031" src="https://user-images.githubusercontent.com/964912/50983556-8a642b00-14b4-11e9-80fd-5a5fe14a83c6.png">

cc @ampinsk

**Comment by ampinsk** [comment]
I actually think this one is fine: 

https://colorable.jxnblk.com/f97583/450c0f

![screen shot 2019-01-10 at 11 12 10 am](https://user-images.githubusercontent.com/10404068/50990961-5bf04b00-14c8-11e9-98a1-211f491836a1.png)

Does someone else want to verify? If not I think we can close this.

**Comment by ampinsk** [comment]
@agisilaos Would you like to verify this one as well?

**Comment by ampinsk** [comment]
I confirmed with the design systems team that passing AA is ✅ so I'm going to close this out.
